NGitLab/IMergeRequestClient.cs

```diff
@@ -58,5 +58,44 @@ public interface IMergeRequestClient
         IMergeRequestApprovalClient ApprovalClient(int mergeRequestIid);
 
         IEnumerable<Issue> ClosesIssues(int mergeRequestIid);
+
+        /// <summary>
+        /// Gets the resource label events.
+        ///
+        /// url like GET /projects/:id/merge_requests/:merge_request_iid/resource_label_events
+        ///
+        /// </summary>
+        /// <param name="projectId">The project id.</param>
+        /// <param name="mergeRequestIid">The id of the merge request in the project's scope.</param>
+        /// <returns>A collection of the resource label events linked to this merge request.</returns>
+        IEnumerable<ResourceLabelEvent> ResourceLabelEvents(int projectId, int mergeRequestIid);
```

Comment:
`GitLabCollectionResponse` implement both `IEnumerable` and `IAsyncEnumerable`.

As they were no prior existing methods, you can likely just let the "async" ones